### PR TITLE
fix(chrome-extension): Ensure synced client cookie URLs resolve appropriately

### PR DESCRIPTION
**Pull request by tmilewski**
## Description

Ensures that there's a valid URL as some don't resolve with a protocol

Fixes SDK-1867

## Checklist

- [ ] `npm test` runs as expected.
- [ ] `npm run build` runs as expected.
- [ ] (If applicable) [JSDoc comments](https://jsdoc.app/about-getting-started.html) have been added or updated for any package exports
- [ ] (If applicable) [Documentation](https://github.com/clerk/clerk-docs) has been updated

## Type of change

- [x] 🐛 Bug fix
- [ ] 🌟 New feature
- [ ] 🔨 Breaking change
- [ ] 📖 Refactoring / dependency upgrade / documentation
- [ ] other:

### CODE REVIEW

**Comment by tmilewski** [comment]
!snapshot

**Comment by clerk-cookie** [comment]
Hey @tmilewski - the snapshot version command generated the following package versions:
| Package | Version |
| --- | --- |
| @clerk/chrome-extension | 1.1.2-snapshot.vd49286e |
| gatsby-plugin-clerk | 5.0.0-beta.45 |

Tip: Use the snippet copy button below to quickly install the required packages.
`@clerk/chrome-extension`
```sh
npm i @clerk/chrome-extension@1.1.2-snapshot.vd49286e --save-exact
```
`gatsby-plugin-clerk`
```sh
npm i gatsby-plugin-clerk@5.0.0-beta.45 --save-exact
```

**Review by dimkl — DISMISSED** [review]
Instead of making this change, why don't we fix the code where it's being called to pass a valid URL with protocol?
This issue is caused by incorrectly passing the `clerk.frontendApi` (which is not a full URL) as param in the JWTHandler.

ref: https://github.com/clerk/javascript/blob/1a704ed9de3ad222ff17d10cb383639b2fc038b9/packages/chrome-extension/src/singleton.ts#L48

**Comment by tmilewski** [comment]
> Instead of making this change, why don't we fix the code where it's being called to pass a valid URL with protocol? This issue is caused by incorrectly passing the `clerk.frontendApi` (which is not a full URL) as param in the JWTHandler.
> 
> ref:
> 
> https://github.com/clerk/javascript/blob/1a704ed9de3ad222ff17d10cb383639b2fc038b9/packages/chrome-extension/src/singleton.ts#L48

@dimkl Yup, I just prepended it. I'm going to leave the other method as there are other cases where that will be necessary and ultimately extended.

File: packages/chrome-extension/src/singleton.ts [hunk 1]
```diff
@@ -45,7 +45,7 @@ export async function buildClerk({
   // Set up cookie params based on environment
   const getClientCookieParams: GetClientCookieParams = isProd
     ? {
-        urls: key.frontendApi,
+        urls: `https://${key.frontendApi}`,
```
Comment (anagstef) [inline packages/chrome-extension/src/singleton.ts:48]: Can we merge the logic we have here and in `ensureFormattedUrl` somehow? 
For example, use here the `ensureFormattedUrl` function.

Comment (tmilewski) [reply packages/chrome-extension/src/singleton.ts:48]: @anagstef `ensureFormattedUrl` is going to handle a lot more use-cases soon. This PR is largely to solve a singular current issue.

**Review by anagstef — APPROVED** [review]
LGTM

**Comment by dimkl** [comment]
> @dimkl Yup, I just prepended it. I'm going to leave the other method as there are other cases where that will be necessary and ultimately extended.

@tmilewski Could you provide example of other cases?
Based on the code, the only case where the url as string is used is for a production instance where we explicitly pass the frontendApiKey (prefixed with `https` after this PR). In the case of the development instance, an array of URLs is passed and this code is not executed.
Could we revert the changes in the `packages/chrome-extension/src/utils/cookies.ts` as it seems unnecessary?

**Pull request merged**